plugins/mysql/mysql_disk_by_user

```diff
@@ -0,0 +1,113 @@
+#!/usr/bin/env python3
+# -*- python -*-
+# -*- coding: utf-8 -*-
+#
+# Plugin to monitor MySQL database disk usage per user
+#
+# Usage: Place in /etc/munin/plugins/ (or link it there using ln -s)
+#   Add this to your /etc/munin/plugin-conf.d/munin-node:
+#       [mysql_disk_size]
+#       user mysql
+#       users_with_underscores foo_bar d_ritchie # usernames of users in the
+#                                                # system that include an underscore
+#       limit 1024000 # minimum db size in order to report a specific user,
```

Comment:
Maybe use `0` as the *no limit* value. This is quite common practice. Additionally it avoids the ambiguity of interpreting a number as as string under certain circumstances.

Additionally it would be good to use a more precise name: `limit` is a bit too broad for its purpose.

Reply:
Done and done.